Content.Server/Voting/Managers/VoteManager.DefaultVotes.cs

```diff
@@ -158,7 +158,13 @@ private void CreatePresetVote(IPlayerSession? initiator)
 
         private void CreateMapVote(IPlayerSession? initiator)
         {
-            var maps = _gameMapManager.CurrentlyEligibleMaps().ToDictionary(map => map, map => map.MapName);
+            var maps = new Dictionary<string, GameMapPrototype>();
+            var eligibleMaps = _gameMapManager.CurrentlyEligibleMaps().ToList();
+            maps.Add(Loc.GetString("ui-vote-secret-map"), _random.Pick(eligibleMaps));
+            foreach (var map in eligibleMaps)
+            {
+                maps.Add(map.MapName, map);
+            }
```

Comment:
Is there something wrong with
```suggestion
            var maps = _gameMapManager.CurrentlyEligibleMaps().ToDictionary(map => map, map => map.MapName);
            maps.Add(Loc.GetString("ui-vote-secret-map"), _random.Pick(eligibleMaps));
```

Reply:
The submitter was probably not aware of ToDictionary(). Committing suggestion.